ios/sensor/ReanimatedSensorContainer.m

```diff
@@ -1,5 +1,7 @@
 #import "ReanimatedSensorContainer.h"
+#if !TARGET_OS_TV
```

Comment:
I believe this won't work on MacOS etc.

Maybe it's better to check `TARGET_OS_IOS` or `TARGET_IPHONE_SIMULATOR``? Obviously this has to compile for iPad, iPod etc.

Or maybe let's check if `<CoreMotion/CoreMotion.h>` header exists (similarly as for JSC/Hermes detection)?